TASVideos/Pages/Forum/Posts/Edit.cshtml.cs

```diff
@@ -233,6 +234,7 @@ public async Task<IActionResult> OnPostDelete()
 			await _publisher.SendForum(
 				post.Topic!.Forum!.Restricted,
 				$"{(topicDeleted ? "Topic" : "Post")} DELETED by {User.Name()}",
+				$"[{(topicDeleted ? "Topic" : "Post")} DELETED]({{0}}) by {User.Name()}",
 				$"{post.Topic!.Forum!.ShortName}: {post.Topic.Title}",
```

Comment:
I think ideally this link would go on `{post.Topic.Title}` but I was hesitant to create *another* variable just for this one use, makes me wonder if there's some better way to design this-- possibly creating a new class to store plaintext and markdown (and maybe HTML/others?) variants of these strings? But then I'm not sure if that'd be too invasive of a change so I dunno, feedback is welcome on this front.